src/Microsoft.DocAsCode.Build.RestApi/Swagger/Internals/JsonLocation/JsonLocationInfo.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.RestApi.Swagger.Internals
+{
+    using Microsoft.DocAsCode.Common;
+
+    internal class JsonLocationInfo
+    {
+        public string FilePath { get; }
+
+        public string JsonLocation { get; }
+
+        public JsonLocationInfo(string swaggerPath, string jsonLocation)
+        {
+            FilePath = PathUtility.NormalizePath(swaggerPath);
+            JsonLocation = jsonLocation;
+        }
+
+        public override bool Equals(object obj)
+        {
+            var other = obj as JsonLocationInfo;
+            if (other == null)
+            {
+                return false;
+            }
+
+            return FilePathComparer.OSPlatformSensitiveStringComparer.Equals(FilePath, other.FilePath)
+                && FilePathComparer.OSPlatformSensitiveStringComparer.Equals(JsonLocation, other.JsonLocation);
```

Comment:
yes you're right, should not ignore case

---
In reply to: [101440772](https://github.com/dotnet/docfx/pull/1308#discussion_r101440772) [](ancestors = 101440772)